src/constants/extendedStaticJsonRpcProvider.ts

```diff
@@ -0,0 +1,53 @@
+import { deepCopy } from '@ethersproject/properties'
```

Comment:
This isn't a constant, it's a class. Can you move all of this to a new folder that deals with rpc, maybe `rpc/StaticJsonRpcProvider.ts`? (should start with a capital, as its a class)